plugins/cache-redis/README.md

```diff
@@ -32,12 +33,11 @@ If no configuration is provided the .amplicationrc.json file will use be used as
 
 ```json
 {
-    "host": "localhost",
-    "port": 6379,
-    "ttl": 5,
-    "max": 100,
```

Comment:
Why was "max" removed if it's still part of the .amplicationrc?

Reply:
good spot, removing it as it's not needed by the new library